spec/controllers/stories_bulk_destroy_controller_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+describe StoriesBulkDestroyController do
+  describe "#create" do
+    context "when receive an array of story ids" do
+      it "destroys stories" do
+        user = create(:user, :with_team)
+        project = create(:project, users: [user], teams: [user.teams.first])
+        story_1 = create(:story, project: project, requested_by: user)
+        story_2 = create(:story, project: project, requested_by: user)
+        story_3 = create(:story, project: project, requested_by: user)
+
+        sign_in user
+
+        post :create, project_id: project.id, story_ids: [story_1.id, story_2.id]
+
+        expect(project.stories).to eq([story_3])
```

Comment:
If we ended up adding different responses depending on the result of this operation, we may also have to test both cases with expectations for the http response as well at a minimal i guess